src/blockchain_utilities/blockchain_import.cpp

```diff
@@ -225,7 +225,7 @@ int check_flush(cryptonote::core &core, std::vector<block_complete_entry> &block
 
     // process block
 
-    block_verification_context bvc = boost::value_initialized<block_verification_context>();
```

Comment:
You're adding this context and then removing it again in this commit. Can you squash this, so you don't touch the lines at all?